ui/packages/consul-ui/app/components/consul/source/README.mdx

```diff
@@ -0,0 +1,20 @@
+# Consul::Source
+
+A presentational component for showing a source name. This is a similar component to `Consul::Kind` and `Consul::ExternalSource`, except `Consul::Source` does not relate to a Service.
+
+```hbs preview-template
+<Consul::Source @source='Source Name' />
+```
+
+### Arguments
+
+| Argument/Attribute | Type | Default | Description |
+| --- | --- | --- | --- |
+| `source` | `string` |  | A string to be passed down and displayed as the source name. |
```

Comment:
I went with string here instead of an object because to make this component more generic. This is easier than scoping all the possible objects that could be received.